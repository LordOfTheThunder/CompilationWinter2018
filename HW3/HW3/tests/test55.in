void TestIf() {
	int t;
	if (t == 3) {
		int x;
		x = 5;
	}
	x = 4;
}
