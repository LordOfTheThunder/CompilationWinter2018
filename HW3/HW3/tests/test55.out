---end scope---
x INT 1
---end scope---
line 7: variable x is not defined
